### ISSUE TITLE
markdown-preview-enhanced packages not working 

### ISSUE THREAD

**Issue by tmfdk333** [shd101wyy/markdown-preview-enhanced#543]
"markdown-preview-enhanced" files and folders in "file_path / .atom / packages /" but "markdown-preview-enhanced" does not work.
I restart the Atom And reinstall from markdown-preview-enhanced gitHub, but it was the same. 

_Atom Version_: 1.18.0
_System_: Mac OS X 10.12.6

**Comment by shd101wyy** [comment]
Hi @tmfdk333 ,

Have you tried installing this package from the atom directly.  [doc is here](https://shd101wyy.github.io/markdown-preview-enhanced/#/installation)  

Thanks

**Comment by tmfdk333** [comment]
@shd101wyy 

I tried it from the Atom directly at first, but the package did not work too. 
The same thing happened when I deleted the package from the atom and reinstalled it.
Thanks!

**Comment by shd101wyy** [comment]
Sorry, what do you mean `not working`? 😂  
You can't install it successfully or you just can't open the preview?   
Can you find the `Markdown Preview Enhanced: Toggle Preview` from the command palette?  
You can also try installing through the terminal `apm install markdown-preview-enhanced` and then check if there is any error happened.

**Comment by tmfdk333** [comment]
Oh, I can install it successfully but can't open the preview. 
I can't find `Markdown Preview Enhanced: Toggle Preview` from the command palette but I can see `packages -> markdown-preview-enhanced -> toggle` in the Atom menu.

Also I check through the terminal, I can see the message in terminal.
`Installing markdown-preview-enhanced to /Users/seul-a/.atom/packages ✓`

**Comment by shd101wyy** [comment]
Can you try reinstalling the package?

```sh
apm uninstall markdown-preview-enhanced
apm install markdown-preview-enhanced
```

Then open developer tool and send a screenshot of the JavaScript console.  
Thanks

**Comment by shd101wyy** [comment]
Also, can you open your developer tool and check the javascript console. When you click `packages -> markdown-preview-enhanced-> toggle`, if there any error pop up?

**Comment by shd101wyy** [comment]
I just released a new version. Dunno if it works.

**Comment by RNCTX** [comment]
worked for a minute.  tried to add custom CSS per your docs and mine has disappeared as well.

completely wiped ~/.atom and reinstalled, and it's still gone.  it'd be nice to know where outside of there you are storing things.

**Comment by shd101wyy** [comment]
@RNCTX config files are stored at `~/.mume` folder.

**Comment by RNCTX** [comment]
okay, wiping out ~/.mume got the window to come back up.

as I said I added (a very long) set of CSS which caused it to break for me.

**Comment by tmfdk333** [comment]
I found out what was the problem. It was because of the wrong CSS about Markdown Preview. 
After modifying the CSS, I was able to see previews that worked fine. Thanks!

**Comment by cbuteau** [comment]
I had the same problem after a reboot.
When I brought up the command-palette I found no markdown preview enhanced.

I found uninstalling and reinstalling it fixed it.

![image](https://user-images.githubusercontent.com/1906839/32616571-9fd0d232-c540-11e7-9e0b-4465b43e8dce.png)

**Comment by shd101wyy** [comment]
@cbuteau how did you install this package?

**Comment by cbuteau** [comment]
Through atom.

**Comment by jboillat** [comment]
An alternative would be to use or install VScode with the markdown-preview-enhanced extension, it is working perfectly on my Ubuntu system

**Comment by cbuteau** [comment]
Yeah that is a solid alternative.  I think both are maintained by the same author.

**Comment by jboillat** [comment]
Yes  it is. I don' t like Microsost  but VScode is definitely better than Atom.On 25 May 2020 18:23, cbuteau <notifications@github.com> wrote:
Yeah that is a solid alternative.  I think both are maintained by the same author.

**Comment by retospect** [comment]
Just had the same issue (preview not showing up after shortcut or menu). 
Do make sure the file extension is .md - and not ,md ...

**Comment by yeadoncty** [comment]
Hey there! Newbie here, I am trying to create my first website with VS and would like to preview so, I installed the Markdown Preview extension but I'm stuck. The shortcuts (Mac) aren't working. Any help would be greatly appreciated.

**Comment by shd101wyy** [comment]
@yeadoncty try right click at the markdown preview, then choose `Markdown Preview Enhanced: Open Preview to the Side` from the context menu.

**Comment by cbuteau** [comment]
Disable the default markdown preview that comes with atom and enable this one then Shift-Ctrl-M launches the preview I think...
